pipelines/scipipe/ap_verify.groovy

```diff
@@ -262,6 +278,7 @@ def void verifyDataset(Map p) {
         )
       }
 
+      
```

Comment:
Extra blank line looks a bit odd.

Reply:
Deleted the blank line, squashed the extra commit and rebased.